### PR TITLE
migration: refine titles about migrating small datasets and large datasets from MySQL to TiDB

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: two-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This section takes the example of a region where two availability zones AZ1 and
 
 The architecture of the cluster deployment is as follows:
 
-- The cluster has four replicas: two Voter replicas in AZ1, one Voter replica, and one Learner replica in AZ2. For the TiKV component, each rack has a proper label.
+- The cluster has six replicas: three Voter replicas in AZ1, two Voter replicas, and one Learner replica in AZ2. For the TiKV component, each rack has a proper label.
```
Comment (dveeden) [inline two-data-centers-in-one-city-deployment.md:25]: I think this belongs to a different PR?

**Review by dveeden — APPROVED** [review]

File: two-data-centers-in-one-city-deployment.md (hunk 1 above)
Comment (hfxsd) [reply two-data-centers-in-one-city-deployment.md:25]: It's also a feedback from AskDocs. So I updated it in the same PR. You can safely ignore it.

File: TOC.md [hunk 2]
```diff
@@ -133,8 +133,8 @@
   - [Import Best Practices](/tidb-lightning/data-import-best-practices.md)
   - Migration Scenarios
     - [Migrate from Aurora](/migrate-aurora-to-tidb.md)
-    - [Migrate MySQL of Small Datasets](/migrate-small-mysql-to-tidb.md)
-    - [Migrate MySQL of Large Datasets](/migrate-large-mysql-to-tidb.md)
+    - [Migrate Small MySQL Datasets](/migrate-small-mysql-to-tidb.md)
+    - [Migrate Large MySQL Datasets](/migrate-large-mysql-to-tidb.md)
```
[the remark this answers is not in the document]
Comment (qiancai) [reply TOC.md:137]: Perhaps, we can keep the title in TOC the same as that in the doc because it is not very long?

File: two-data-centers-in-one-city-deployment.md (hunk 1 above)
Comment (qiancai) [inline two-data-centers-in-one-city-deployment.md:25]:
```suggestion
- The cluster has six replicas: three Voter replicas in AZ1, and two Voter replicas along with one Learner replica in AZ2. For the TiKV component, each rack has a proper label.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM
